src/main/java/com/opentok/Connect.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * OpenTok Java SDK
+ * Copyright (C) 2022 Vonage.
+ * http://www.tokbox.com
+ *
+ * Licensed under The MIT License (MIT). See LICENSE file for more information.
+ */
+package com.opentok;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents an Audio Stream connection response.
+ */
+@JsonIgnoreProperties(ignoreUnknown=true)
+public class Connect {
+
+	@JsonProperty private String id;
+	@JsonProperty private String connectionId;
+
+	protected Connect() {}
+
+	/**
+	 * Do not call this method. To initiate a connection, call the
+	 * {@link OpenTok#connectAudioStream(String, String, ConnectProperties)} method.
+	 */
```

Comment:
Does this method have to be public? I think the docs could be clearer: "To initiate an audio stream connection to a session...."

Maybe the classes should be named AudioStreamerConnection and AudioStreamerConnectionProperties. "Connection" seems like an overloaded word in the realm of OpenTok (e.g.  https://tokbox.com/developer/guides/basics/#connecting-session).

Reply:
I think the docs could be clearer: "To initiate an audio stream connection to a session...."

Also, maybe the classes should be named AudioStreamerConnection and AudioStreamerConnectionProperties. "Connection" seems like an overloaded word in the realm of OpenTok (e.g.  https://tokbox.com/developer/guides/basics/#connecting-session).